canopen_core/src/lifecycle_manager.cpp

```diff
@@ -70,20 +70,20 @@ void LifecycleManager::init(std::shared_ptr<ros2_canopen::ConfigurationManager>
 
 bool LifecycleManager::load_from_config()
 {
-  std::vector<std::string> devices;
-  uint32_t count = this->config_->get_all_devices(devices);
+  std::vector<std::string> device_names;
+  uint32_t count = this->config_->get_all_devices(device_names);
   RCLCPP_INFO(this->get_logger(), "Configuring for %u devices.", count);
 
   // Find master in configuration
-  for (auto it = devices.begin(); it != devices.end(); it++)
+  for (const auto & device_name : device_names)
   {
-    uint8_t node_id = config_->get_entry<uint8_t>(*it, "node_id").value();
-    std::string change_state_client_name = *it;
-    std::string get_state_client_name = *it;
+    uint8_t node_id = config_->get_entry<uint8_t>(device, "node_id").value();
```

Comment:
CI:

  /root/target_ws/src/ros2_canopen/canopen_core/src/lifecycle_manager.cpp:80:51: error: ‘device’ was not declared in this scope

Not sure how this happens as the code looks correct...